gravatar-quickeditor/src/test/java/com/gravatar/quickeditor/data/repository/AvatarRepositoryTest.kt

```diff
@@ -54,51 +50,34 @@ class AvatarRepositoryTest {
 
         val result = avatarRepository.getAvatars(email)
 
-        assertEquals(Result.Failure<IdentityAvatars, QuickEditorError>(QuickEditorError.TokenNotFound), result)
+        assertEquals(Result.Failure<EmailAvatars, QuickEditorError>(QuickEditorError.TokenNotFound), result)
     }
 
     @Test
     fun `given token stored when avatar service fails then Failure result`() = runTest {
         coEvery { tokenStorage.getToken(any()) } returns "token"
 
-        coEvery { avatarService.retrieveCatching(any()) } returns Result.Failure(ErrorType.SERVER)
-        coEvery { identityService.retrieveCatching(any(), any()) } returns Result.Success(createIdentity("1"))
+        coEvery { avatarService.retrieveCatching(any(), any()) } returns Result.Failure(ErrorType.SERVER)
 
         val result = avatarRepository.getAvatars(email)
 
         assertEquals(
-            Result.Failure<IdentityAvatars, QuickEditorError>(QuickEditorError.Request(ErrorType.SERVER)),
+            Result.Failure<EmailAvatars, QuickEditorError>(QuickEditorError.Request(ErrorType.SERVER)),
             result,
         )
     }
 
     @Test
-    fun `given token stored when identity service fails then Failure result`() = runTest {
-        coEvery { tokenStorage.getToken(any()) } returns "token"
-
-        coEvery { avatarService.retrieveCatching(any()) } returns Result.Success(listOf(createAvatar("1")))
-        coEvery { identityService.retrieveCatching(any(), any()) } returns Result.Failure(ErrorType.SERVER)
-
-        val result = avatarRepository.getAvatars(email)
-
-        assertEquals(
-            Result.Failure<IdentityAvatars, QuickEditorError>(QuickEditorError.Request(ErrorType.SERVER)),
-            result,
-        )
-    }
-
-    @Test
-    fun `given token stored when avatar service and identity service succeed then Success result`() = runTest {
+    fun `given token stored when avatar service succeed then Success result`() = runTest {
```

Comment:
⛏ We can specify the action in the method declaration. Maybe, something like:

```suggestion
    fun `given token stored when fetch avatars succeed then Success result`() = runTest {
```